lib/serverless/api-gateway.js

```diff
@@ -54,7 +54,10 @@ class LambdaProxyWebResponse {
  * @returns {Object<string, string>} The normalized query string map
  */
 function normalizeQueryStringParameters(event) {
-  if (!event.multiValueQueryStringParameters) {
+  if (
+    !event.multiValueQueryStringParameters ||
+    Object.keys(event.multiValueQueryStringParameters).length === 0
```

Comment:
I don't understand. If we have `multiValueQueryStringParameters` defined at all, we should not have `queryStringParameters` defined. So why would we return `queryStringParameters` if the multi valued property is defined with zero elements?

Reply:
In the sample event captured by the customer, both `queryStringParameters` and `multiValueQueryStringParameters` were defined, event though each had zero properties. I could see changing this to check for properties in each. The definition of one, though, doesn't exclude the definition of the other, based on the input they've captured.